kolibri/plugins/setup_wizard/assets/src/state/store.js

```diff
@@ -1,17 +1,59 @@
 import Vuex from 'kolibri.lib.vuex';
+import { currentLanguage } from 'kolibri.utils.i18n';
+import { findKey } from 'lodash';
+import { permissionPresets } from './constants';
 import {
   initialState as coreInitialState,
   mutations as coreMutations,
 } from 'kolibri.coreVue.vuex.store';
 
 const initialState = {
-  pageState: {
-    submitted: false,
+  onboardingData: {
+    language_id: currentLanguage,
+    facility: {
+      name: '',
+    },
+    superuser: {
+      full_name: '',
+      username: '',
+      password: '',
+    },
+    preset: findKey(permissionPresets, preset => preset.default) || '',
   },
+  loading: false,
+  error: false,
+  onboardingStep: 1,
 };
+
 const mutations = {
-  SET_SUBMITTED_STATE(state, submittedFlag) {
-    state.pageState.submitted = submittedFlag;
+  SET_LANGUAGE(state, language_id) {
+    state.onboardingData.language_id = language_id;
+  },
+  SET_FACILITY_NAME(state, name) {
+    state.onboardingData.facility.name = name;
+  },
+  SET_SU(state, { name, username, password }) {
```

Comment:
@rtibbles

Reply:
Could probably simplify this to a single `superuser` argument and just set `state.onboardingData.superuser = superuser;` but this is also nicely explicit.